pylint/checkers/format.py

```diff
@@ -524,15 +525,15 @@ def visit_default(self, node: nodes.NodeNG) -> None:
         except AttributeError:
             tolineno = node.tolineno
         assert tolineno, node
-        lines = []
+        lines: list[str] = []
         for line in range(line, tolineno + 1):
             self._visited_lines[line] = 1
             try:
                 lines.append(self._lines[line].rstrip())
             except KeyError:
                 lines.append("")
 
-    def _check_multi_statement_line(self, node, line):
+    def _check_multi_statement_line(self, node: nodes.NG, line: str) -> None:
```

Comment:
```suggestion
    def _check_multi_statement_line(self, node: nodes.NodeNG, line: str) -> None:
```

?

Reply:
Nice catch ! I always forget that as long as astroid do not expose type we can put anything here and mypy is happy with really wrong typing...